iotile_ext_cloud/iotile/cloud/cloud.py

```diff
@@ -71,6 +74,10 @@ def __init__(self, domain=None, username=None, **kwargs):
         self.token = self.api.token
         self.token_type = self.api.token_type
 
```

Comment:
I would recommend renaming this property to make it a little less verbose, something like `verify_server`.  Originally I was wondering why it was a property at all but I'm guessing that's to ensure that users don't try to set it on their own, which makes sense.